reference/service/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Pulumi Cloud Console
+---
+
+[The Pulumi Cloud Console](https://app.pulumi.com) allows you to manage your stacks online. It enables
+you to to collaborate with multiple developers, protect against concurrent updates, store resource
+history, integrate with your CI/CD system, and more!
+
+## Getting Started
+
+To get started using the Pulumi Cloud Console, go to [app.pulumi.com](https://app.pulumi.com) and
+sign in with your GitHub account.
+
+> **NOTE**
+>
+> Support for more kinds of identity providers is coming soon. If you require something
+> other than GitHub, please [let us know](https://www.pulumi.com/about/#contact-us).
+
+## Organizations
+
+The Pulumi Cloud Console manages stacks by placing them within _Organizations_.
+
+When you first log in, a Pulum organization is created with the same name as your GitHub account.
```

Comment:
I wouldn't describe it this way. This is how the implementation works, but leads to a confusing mental model. We should instead talk about Users and Organizations. We recently discussed in a PR with @justinvp some terminology we can use to help clarify this.